app/models/etl/feedex.rb

```diff
@@ -0,0 +1,66 @@
+class ETL::Feedex
+  def self.process(*args)
+    new(*args).process
+  end
+
+  def initialize(date:)
+    @date = date
+  end
+
+  def process
+    extract_events
+    load_metrics
+  end
+
+private
+
+  BATCH_SIZE = 10_000
+
+  def extract_events
+    feedex_service.find_in_batches do |events|
+      Events::Feedex.import(events, batch_size: BATCH_SIZE)
+    end
+  end
+
+  def load_metrics
+    conn = ActiveRecord::Base.connection
+    date_to_s = date.strftime("%F")
+
+    conn.execute(load_metrics_query(date_to_s))
+    conn.execute(clean_up_query)
+  end
+
+  def load_metrics_query(date_to_s)
+    <<~SQL
+      UPDATE facts_metrics
+      SET number_of_issues = s.number_of_issues
+      FROM (
+        SELECT base_path,
+               number_of_issues,
+               dimensions_items.id
+        FROM events_feedexes, dimensions_items
+        WHERE page_path = base_path AND latest = 'true'
+      ) AS s
+      WHERE dimensions_item_id = s.id AND dimensions_date_id = '#{date_to_s}'
```

Comment:
Could you please check https://github.com/alphagov/content-performance-manager/pull/524 and update accordingly? We should not rely on `latest` for the queries.